File: salimmo-frontend/src/app/models/propertyenergies/propertyEnergiesDto.ts
import {PropertyDto} from "../property/propertyDto";
import {HeatingType} from "../enums/heating-type";

export class PropertyEnergiesDto {
  id!:number;
  property!:PropertyDto;
  heatingType!:HeatingType;
  hotWater!:string;
  electricalInstallation!:string;
  heatingLoads!:number;
  waterLoads!:number;
  electricLoads!:number;
}

File: salimmo-parent/property-service/src/main/java/org/propertyservice/dto/PropertyEnergiesDto.java
package org.propertyservice.dto;


import lombok.*;
import org.propertyservice.enums.HeatingType;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class PropertyEnergiesDto {
    private Long id;
    private PropertyDto property;
    private HeatingType heatingType;
    private String hotWater;
    private String electricalInstallation;
    private Double heatingLoads;
    private Double waterLoads;
    private Double electricLoads;
}
